UnitTests/BraintreeCoreTests/Analytics/FPTIBatchData_Tests.swift

```diff
@@ -99,6 +101,8 @@ final class FPTIBatchData_Tests: XCTestCase {
         XCTAssertNil(eventParams[1]["error_desc"])
         XCTAssertEqual(eventParams[0]["correlation_id"] as? String, "fake-correlation-id-1")
         XCTAssertNil(eventParams[1]["correlation_id"])
+        XCTAssertEqual(eventParams[0]["is_vault"] as? Bool, true)
+        XCTAssertNil(eventParams[1]["is_vault"])
```

Comment:
Based on the above, I would expect this test is failing currently. Looks like there is an error with `MockAPIClient` not overriding the `sendAnalyticsEvent` properly that is blocking the test execution currently.

Reply:
Thanks, looks like the issue was that the `isVaultRequest` parameter wasn't in the right order and `MockAPIClient` didn't have the updated `request` parameter name